openshift/release/resolve.sh

```diff
@@ -24,10 +24,10 @@ function resolve_resources(){
         -e "s+ko://++" \
         -e "s+eventing.knative.dev/release: devel+eventing.knative.dev/release: ${release}+" \
         -e "s+app.kubernetes.io/version: devel+app.kubernetes.io/version: ${release}+" \
-        -e "s+knative.dev/eventing/cmd/broker/ingress+${image_prefix}mtbroker-ingress${image_tag}+" \
-        -e "s+knative.dev/eventing/cmd/broker/filter+${image_prefix}mtbroker-filter${image_tag}+" \
-        -e "s+knative.dev/eventing/cmd/mtbroker/ingress+${image_prefix}mtbroker-ingress${image_tag}+" \
-        -e "s+knative.dev/eventing/cmd/mtbroker/filter+${image_prefix}mtbroker-filter${image_tag}+" \
+        -e "s+knative.dev/eventing/cmd/broker/ingress+${image_prefix}ingress${image_tag}+" \
+        -e "s+knative.dev/eventing/cmd/broker/filter+${image_prefix}filter${image_tag}+" \
+        -e "s+knative.dev/eventing/cmd/mtbroker/ingress+${image_prefix}ingress${image_tag}+" \
+        -e "s+knative.dev/eventing/cmd/mtbroker/filter+${image_prefix}filter${image_tag}+" \
```

Comment:
I believe `mtbroker` matching rule can be completely removed. I'd rather get ACK first @pierDipi.

Reply:
Yes, there is `mtbroker` in `cmd/` https://github.com/openshift-knative/eventing/tree/release-next/cmd